src/data-sources/intermine/intermine.api.js

```diff
@@ -143,7 +143,7 @@ class IntermineAPI extends RESTDataSource {
   }
 
   // get an ordered, paginated list of genes
-  async getGenes({organism, family, start=0, size=10}={}) {
+  async getGenes(organism=undefined, family=undefined, start=0, size=10) {
```

Comment:
This shouldn't have been changed. The previous pattern is how you define named parameters in JavaScript, allowing you to provide arguments for an arbitrary subset of the parameters, whereas the new code uses positional parameters, meaning, for example, if you want to filter by family then you also have to provide an organism argument.

This breaks the genes entrypoint if you try to filter by family.